README.md

```diff
@@ -41,6 +41,11 @@ Developers are expected to run the app locally using [docker-compose](https://do
 * `make build` / `docker compose build` will rebuild the Funding Service image.
 * `make clean-build` / `docker compose build --no-cache` will rebuild the Funding Service image, bypassing caching. This should rarely be needed.
 
+### SSO
+By default, local development and locally-run end to end tests use a [stub server](./stubs/sso/) to implement the same  interface as Microsoft SSO flow. This is so we can reliably run e2e tests locally without hitting the real SSO endpoints.
```

Comment:
It's also so that devs don't need our test azure AD credentials to be able to use the app, making initial setup easier.